templates/expo-template-bare-minimum/ios/HelloWorld.xcodeproj/project.pbxproj

```diff
@@ -433,6 +476,7 @@
 				COPY_PHASE_STRIP = YES;
 				ENABLE_NS_ASSERTIONS = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = i386;
```

Comment:
do we need this?

Reply:
Probably no, but the project generated by `expo prebuild` also has this setting, so it won't break anything.